src/App.style.js

```diff
@@ -0,0 +1,54 @@
+import styled from 'styled-components'
+
+/** image */
+import cthulhu_img from './assets/images/cthulhu_img.jpg'
+
+export const AppContainer = styled.div`
+  color: #dfdfdf;
+  display: flex;
+  flex-flow: column nowrap;
+  height: 100vh;
+  width: 100%;
+  margin: 0;
+  padding: 0;
+  justify-content: center;
+  align-items: center;
+  background-image: url(${cthulhu_img});
```

Comment:
Background image doesn't resize with the browser and would need to incorporate the reveal work being done. Not sure I'm sold on this, but open to the idea if you have ideas/fixes.

Reply:
I changed the property background-size, from `cover` to `contain`, could you check it out, please?

About the asterisks, could you print the problem for me? Because here I can see the letters that way.

![image](https://user-images.githubusercontent.com/13955181/95131587-af8ed780-0734-11eb-9e64-a24c13df39db.png)

Reply:
Looking now, sorry for the delay.

Reply:
So it's a little hard to see, but there are individual lines beneath the asterisks in that picture you posted. I had the asterisks centered above each individual line like this
![image](https://user-images.githubusercontent.com/6750696/95283229-22b94c00-0810-11eb-9a27-d83764d5dd4b.png)

Reply:
Without centering it looks like this with letters, but I want to keep the separation as it's part of the whole guess one letter at a time feel to the game
![image](https://user-images.githubusercontent.com/6750696/95283425-a410de80-0810-11eb-9642-5599b868325d.png)

Reply:
For the cover sizing change it looks a bit strange in mobile: 
![image](https://user-images.githubusercontent.com/6750696/95284139-45e4fb00-0812-11eb-89e8-bc827d3ec980.png)
At a certain breakpoint I'd suggest changing the layout to make this all fit a bit better together.

Reply:
Right, in `App.style.js` line `21`we have that breakpoint, what do you suggest for a mobile layout? I just added position top to make image more visible, but we can do everything here!

Reply:
Well, I did some changes in letter align, could you check please?